modules/cli/pages/podman.adoc

```diff
@@ -44,6 +44,12 @@ To switch back to target Kubernetes sites: `skupper switch kubernetes`
 
 * The latest `skupper` CLI is installed.
 * Podman is installed, see https://podman.io/
++
+--
+By default, Podman v4 uses Netavark which works with Skupper.
+If you are using CNI, for example, if you upgrade from Podman v3, you must also install https://www.cni.dev/plugins/current/[containernetworking-plugins].
+For example, `dnf install containernetworking-plugins` or `apt-get install containernetworking-plugins`.
```

Comment:
@pwright I have validated the package name and the one I have provided you earlier is
incorrect. The correct one is: "podman-plugins".

Reply:
updated, but note that podman-plugins isn't available for a typical ubuntu install.. On the other hand latest ubuntu has podman v4

Reply:
I believe that podman v4 is only available on Ubuntu through the test/unstable channels at the moment.
But maybe it is better to add a note about this package being valid for Fedora, RHEL and CentOS.